src/main/java/gregtech/api/metatileentity/multiblock/MultiblockControllerBase.java

```diff
@@ -375,6 +386,10 @@ public void checkStructurePattern() {
         }
     }
 
+    protected <T> boolean checkAbilityPart(MultiblockAbility<T> ability, BlockPos pos) {
```

Comment:
This is never overridden by anything? What is its intended use, and can that be added to javadocs? Also, do we have anything that needs to override this?

Reply:
see https://github.com/GregTechCEu/GregTech/pull/2420#discussion_r1671788079